docs/prose/source/api.rst

```diff
@@ -0,0 +1,8 @@
+=============
+API Reference
+=============
+
+.. toctree::
+
+   solid-client API <https://docs.inrupt.com/client-libraries/api/javascript/solid-client>
```

Comment:
Do you know if there is a way to use `:apisolidclient:` here as well, so that the link is also correct in the Preview deployment? (No biggie if not; just decreases the likelihood of confusion in the future.)

Reply:
Unfortunately, I don't think we can in a toctree.